### ISSUE TITLE
RGB colours to console via Bukkit are corrupted

### ISSUE THREAD

**Issue by amalon** [KyoriPowered/adventure-platform#38]
RGB colours don't seem to be converted to the named colours when outputting to the console, resulting in corruption.

Gradle dependencies:
implementation "net.kyori:adventure-api:4.3.0"
implementation "net.kyori:adventure-platform-bukkit:4.0.0-SNAPSHOT"

Server:
papermc-1.16.4+b259-1 on archlinux

E.g.:

```java
import net.kyori.adventure.text.Component;
import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
import net.kyori.adventure.text.format.TextColor;
import net.kyori.adventure.text.format.NamedTextColor;
import net.kyori.adventure.audience.Audience;
import net.kyori.adventure.platform.bukkit.BukkitAudiences;
...
    Component named = Component.text("")
                        .append(Component.text("red").color(NamedTextColor.RED))
                        .append(Component.text(" green").color(NamedTextColor.GREEN))
                        .append(Component.text(" blue").color(NamedTextColor.BLUE));
    Component rgb = Component.text("")
                        .append(Component.text("red").color(TextColor.color(0xff0000)))
                        .append(Component.text(" green").color(TextColor.color(0x00ff00)))
                        .append(Component.text(" blue").color(TextColor.color(0x0000ff)));

    BukkitAudiences audiences = BukkitAudiences.create(this);
    Audience consoleAudience = audiences.console();
    consoleAudience.sendMessage(named);
    consoleAudience.sendMessage(rgb);

    getLogger().info(LegacyComponentSerializer.legacySection().serialize(named));
    getLogger().info(LegacyComponentSerializer.legacySection().serialize(rgb));
```
There are 4 outputs here:
1) sendMessage(named): works fine
2) sendMessage(rgb): gets corrupted (though fine in client when sent to player)
3) LegacyComponentSerializer named: works fine
4) LegacyComponentSerializer rgb: works fine

![Screenshot_20201222_231004](https://user-images.githubusercontent.com/210303/102941527-d98da580-44aa-11eb-8b04-3150dfe9e588.png)

**Comment by jmanpenilla** [comment]
This is a paper bug, not an adventure bug. See this for more context: https://github.com/PaperMC/Paper/pull/4221

**Comment by zml2008** [comment]
Fixed with PaperMC/Paper#5205
